README.md

```diff
@@ -218,6 +218,22 @@ bin/slider flex presto1 --component WORKER 2
 
 Please note that if your cluster already had 3 WORKER nodes running, the above command will destroy one of them and retain 2 WORKERs.
 
+# Debugging and Logging
+
+It is recommended that log aggregation of YARN application log files be enabled in YARN, using yarn.log-aggregation-enable property in your yarn-site.xml. Then slider logs created during the launch of Presto-YARN will be available locally on your nodemanager nodes under /var/log/hadoop-yarn/application_<id>/container_<id>/. For any retries attempted by Slider to launch Presto a new container will be launched and hence you will find a new container_<id> directory.  You can look for any errors under errors_*.txt there, and also there is a slider-agent.log file which will give you Slider application lifetime details.
```

Comment:
I think `/var/log/hadoop-yarn` depends on hadoop installation, it may vary from installation to installation.